src/Client/Connection.cpp

```diff
@@ -157,7 +158,7 @@ void Connection::connect(const ConnectionTimeouts & timeouts)
 #else
                 TCP_KEEPIDLE  // __APPLE__
```

Comment:
It is interesting that there is an __APPLE__ preprocessor conditional in the comments for TCP_KEEPIDLE, when it seems like it should be near TCP_KEEPALIVE :)